tests/unix/test_lwt_unix.cppo.ml

```diff
@@ -677,11 +677,28 @@ let bind_tests = [
         (fun () -> Lwt_unix.close socket));
 ]
 
+let dir_tests = [
```

Comment:
Ideally we would have another test such as

```
    test "get other cwd"
    (fun () ->
      Lwt_unix.getcwd () >>= fun here ->
      Lwt_unix.chdir somewhere >>= fun () ->
      Lwt_unix.getcwd () >>= fun there ->
      Lwt_unix.chdir there >>= fun () ->
      Lwt.return (here = there)
    );
```

But:

1. How to choose `somewhere`?
2. How to deal with failures which leave the rest of the tests in an unexpected directory?

I think there are things in setup.ml about directories to run tests in. I'll investigate.